src/current/v24.1/ui-networking-dashboard.md

```diff
@@ -63,6 +111,44 @@ A healthy RPC connection is one that is “bidirectionally connected” and "hea
 
 Metric: `rpc.connection.unhealthy` Gauge of current connections in an unhealthy state (not bidirectionally connected or heartbeating)
 
+## Proxy requests
+
+- In the node view, the graph shows the number of proxy attempts each [gateway node]({% link {{ page.version.version }}/architecture/life-of-a-distributed-transaction.md %}#gateway) is initiating.
+
+- In the cluster view, the graph shows the number of proxy attempts each [gateway node]({% link {{ page.version.version }}/architecture/life-of-a-distributed-transaction.md %}#gateway) in the cluster is initiating.
+
+Metric: `distsender.rpc.proxy.sent` Number of attempts by a gateway to proxy a request to an unreachable [leaseholder]({% link {{ page.version.version }}/architecture/overview.md %}#leaseholder) via a [follower]({% link {{ page.version.version }}/architecture/reads-and-writes-overview.md %}#write-scenario) [replica]({% link {{ page.version.version }}/architecture/overview.md %}#replica).
```

Comment:
Are both the "follower" and "replica" links needed here? I think they will be easy to miss on the page since it will scan as a single linked phrase. Is one more important?